panel/io/resources.py

```diff
@@ -40,21 +41,51 @@ def conffilter(value):
 
 # Handle serving of the panel extension before session is loaded
 RESOURCE_MODE = 'server'
-PANEL_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+PANEL_DIR = Path(__file__).parent.parent
+DIST_DIR = PANEL_DIR / 'dist'
+ASSETS_DIR = PANEL_DIR / 'assets'
 BASE_TEMPLATE = _env.get_template('base.html')
 DEFAULT_TITLE = "Panel Application"
 JS_RESOURCES = _env.get_template('js_resources.html')
 CDN_DIST = f"https://unpkg.com/@holoviz/panel@{js_version}/dist/"
 LOCAL_DIST = "static/extensions/panel/"
-DIST_DIR = Path(__file__).parent.parent / 'dist'
 
-extension_dirs['panel'] = os.path.abspath(os.path.join(PANEL_DIR, 'dist'))
+extension_dirs['panel'] = str(DIST_DIR)
+
+
+def loading_css():
+    from ..config import config
+    with open(ASSETS_DIR / f'{config.loading_spinner}_spinner.svg', encoding='utf-8') as f:
+        svg = f.read().replace('\n', '').format(color=config.loading_color)
+    b64 = b64encode(svg.encode('utf-8')).decode('utf-8')
+    return f"""
+    .bk.pn-loading.arcs:before {{
+      background-image: url("data:image/svg+xml;base64,{b64}")
+    }}
+    .bk.pn-loading.arcs:before {{
```

Comment:
I had to add this css to get the loading indicator working in the notebook example I just added @philippjfr 

I think it should maybe go to the widgets.css file and an updated version should be included in the dark.css files for it to look nice there.

Reply:
This definitely should not be needed. I'll remove it again.

Reply:
I probably did something wrong.

Reply:
Forgot to push `panel/dist/css/loading.css`